### DISCUSSION TITLE
Enabling GPIOs, Sound stops, but PhonieBox keeps in Playing

### DISCUSSION THREAD

**Discussion by gsandreas** [MiczFlor/RPi-Jukebox-RFID#1673] [Q&A]
Hi, 

I have a strance problem.

PhonieBox is playing some Music. When I'm editing the GPIO configuration file ~/RPi-Jukebox-RFID/settings/gpio_settings.ini just flippin one entry from False to True and restart the service by using sudo systemctl restart phoniebox-gpio-control the Sound stops but the time stamp increments.  

I could it got to work again, by coping the BAK file over the gpio_settings.ini or Flip the entry back to False. Whats wrong? I do not get the point, where i can search further... 

Please help me.

**Comment by Lippsson** [comment]
Hi! Which parameter do you want to change, and why? How does your gpio_settings.ini looks like before and after the changes? 
/Andreas

**Reply by gsandreas** [reply]
Any, it was independent from the edit. But i figured out, how it works. I checked, which Port is used and how. by typing 

`raspi-gpio get`

it lists all Port Usages. By picking only Ports, which are configured as Imput, it works. finally.
